### ISSUE TITLE
Add NVIDIA H100 support

### ISSUE THREAD

**Issue by GitHub30** [googlecolab/colabtools#2694]
**Is your feature request related to a problem? Please describe.**
I'm always frustrated when training transfomer takes a long time.

**Describe the solution you'd like**
Add NVIDIA H100 to accelerate training with Transformer Engine.

**Comment by cperry-goog** [comment]
Thanks for the request, in the meantime you can always explore our GCP Marketplace offering and look into acquiring H100s there: https://research.google.com/colaboratory/marketplace.html

**Comment by faruknane** [comment]
Yes, it would be really great for the community to develop faster with high end GPU cards @cperry-goog
